minidump/src/minidump.rs

```diff
@@ -2041,7 +2171,7 @@ impl<'a> MinidumpLinuxMapInfo<'a> {
   base_address          = {:#x}
   final_address         = {:#x}
   kind                  = {:#?}
-  permissions           = 
+  permissions           =
```

Comment:
```suggestion
  permissions           = 
```

This whitespace is intentional